ethcore/src/executive.rs

```diff
@@ -1614,6 +1614,62 @@ mod tests {
 		assert_eq!(state.storage_at(&contract_address, &H256::from(&U256::zero())).unwrap(), H256::from(&U256::from(0)));
 	}
 
+	evm_test!{test_eip1283: test_eip1283_int}
+	fn test_eip1283(factory: Factory) {
+		let x1 = Address::from(0x1000);
+		let x2 = Address::from(0x1001);
+		let y1 = Address::from(0x2001);
+		let y2 = Address::from(0x2002);
+
+		let mut state = get_temp_state_with_factory(factory.clone());
+		state.new_contract(&x1, U256::zero(), U256::from(1)).unwrap();
```

Comment:
The `new_contract` does not seems to be mandatory, it contradicts the comment of `init_code`, but was use this way in other test. Let's keep it.

Reply:
Yeah, this is just to test whether checkpointing is working with `new_contract`. In reality we would not encounter `new_contract` called on an existing account.

Reply:
Ah sorry, I was confused on this and the other test: In this case, we want to create new contracts and init code for `x1`, `x2`, `y1`, `y2`. `new_contract` must be called before `init_code`.